src/components/AddSample/AddSample.jsx

```diff
@@ -0,0 +1,378 @@
+
+const useStyles = makeStyles((theme) => ({
+  inputs: {
+   margin: theme.spacing(2),
+  },
+  formControl: {
+    margin: theme.spacing(2),
+    minWidth: 180,
+  },
+}));
+
+
+function AddSample() {
+  // material ui
+  const classes = useStyles();
+
+  // set stuff so we can use them
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  // get state from redux store
+  const user = useSelector(store => store.user);
+
+  // local states for input fields
+  const [ingredientName, setName] = useState('');
+  const [lotNumber, setLotNumber] = useState('');
+  const [format, setFormat] = useState('');
+  const [ingredientAmount, setAmount] = useState('');
+  const [ingredientUnit, setIngredientUnit] = useState('');
+  const [purity, setPurity] = useState('');
+  const [dateManufactured, setDate] = useState('');
+  const [extractionMethod, setMethod] = useState('');
+  const [city, setCity] = useState('');
+  const [state, setState] = useState('');
+  const [country, setCountry] = useState('');
+  const [harvestDate, setHarvestDate] = useState('');
+  const [sustainability, setSustainability] = useState('');
+  const [cropStrain, setCropStrain] = useState('');
+
+  // functions
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log("form submit")
+  };
+
+  // on shipping button click
+  const shipping = (event) => {
+    event.preventDefault();
+    console.log('shipping info');
+
+    if (
+      !ingredientName ||
+      !lotNumber ||
+      !format ||
+      !ingredientAmount ||
+      !dateManufactured ||
+      !extractionMethod
+    ) {
+      alert("Please complete required inputs");
+    } else {
+      const companyID = user.companyID;
+  
+      dispatch({
+        type: 'SEND_ORDER_INFO',
+        payload: {
+          companyID,
+          ingredientName,
+          ingredientAmount,
+          ingredientUnit,
+          format,
+          purity,
+          dateManufacture,
+          lotNumber,
+          extractionMethod,
+          city,
+          state,
+          country,
+          harvestDate,
+          cropStrain,
+          Sustainability,
+          // orderId
+        }
+      })
+    }
+  }; // end shipping
+
+  const cancel = (event) => {
+    console.log("cancel")
+    setName('');
+    setLotNumber('');
+    setFormat('');
+    setPurity('');
+    setDate('');
+    setMethod('');
+    setCity('');
+    setState('');
+    setCountry('');
+    setCropStrain('');
+    setHarvestDate('');
+    setSustainability('');
+  }; // end cancel
+
+  return (<>
+    <div>
+      <form onSubmit={handleSubmit}>
+        <div>
+        <TextField
+            className={classes.inputs}
+            required
+            label='Ingredient Name'
+            variant='filled'
+            value={ingredientName} 
+            type="text" 
+            onChange={(event) => setName(event.target.value)}/>
+          {/* <input
```

Comment:
Try to get out of the habit of leaving commented out code hanging around.
Don't worry about "losing" old code -- that's what we have git for!